site/layouts/partials/nav.html

```
@@ -0,0 +1,18 @@
+    <div class="logo-icon">T</div>
+    <span>tracker</span> <span class="accent">by 2389.ai</span>
+  </a>
+  <ul class="nav-links">
+    {{ $current := .RelPermalink }}
+    {{ range site.Data.nav.items }}
+      {{ $url := .url }}
+      {{ if not .external }}{{ $url = .url | relURL }}{{ end }}
+      <li>
+        <a href="{{ $url }}"
+           {{ if eq $current (.url | relURL) }}class="active"{{ end }}
+           {{ if .external }}target="_blank"{{ end }}>{{ .label }}</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)